ios/lib/ReactNativeCameraKit/CKCamera.m

```diff
@@ -917,44 +941,40 @@ - (void)createCustomFramesForView:(UIView *)frameView {
 }
 
 - (void)addVisualEffects:(CGRect)inputRect {
-    UIView *topView = [[UIView alloc] initWithFrame:CGRectMake(0, 0, self.frame.size.width, inputRect.origin.y)];
-    topView.backgroundColor = [UIColor colorWithRed:0.0/255.0 green:0.0/255.0 blue:0.0/255.0 alpha:0.4];
-    [self addSubview:topView];
-    
-    UIView *leftSideView = [[UIView alloc] initWithFrame:CGRectMake(0, inputRect.origin.y, self.frameOffset, self.heightFrame)]; //paddingForScanner scannerHeight
-    leftSideView.backgroundColor = [UIColor colorWithRed:0.0/255.0 green:0.0/255.0 blue:0.0/255.0 alpha:0.4];
-    [self addSubview:leftSideView];
-    
-    UIView *rightSideView = [[UIView alloc] initWithFrame:CGRectMake(inputRect.size.width + self.frameOffset, inputRect.origin.y, self.frameOffset, self.heightFrame)];
-    rightSideView.backgroundColor = [UIColor colorWithRed:0.0/255.0 green:0.0/255.0 blue:0.0/255.0 alpha:0.4];
-    [self addSubview:rightSideView];
-    
-    UIView *bottomView = [[UIView alloc] initWithFrame:CGRectMake(0, inputRect.origin.y + self.heightFrame, self.frame.size.width,
-                                                                  self.frame.size.height - inputRect.origin.y - self.heightFrame)];
-    bottomView.backgroundColor = [UIColor colorWithRed:0.0/255.0 green:0.0/255.0 blue:0.0/255.0 alpha:0.4];
-    [self addSubview:bottomView];
-    [self bringSubviewToFront:self.perfomanceBackground];
+    self.overlayView = [[UIView alloc] initWithFrame:self.bounds];
+    self.overlayView.alpha = 0.6;
```

Comment:
Can we define this default magic number on top of the file with capital letters and use it?

Reply:
change on defined constant defaultValue